src/platforms/php/guides/symfony/index.mdx

```diff
@@ -17,32 +17,106 @@ Install the `sentry/sentry-symfony` package:
 composer require sentry/sentry-symfony
 ```
 
-### Configure
+<Alert level="warning" title="Note">
+**Warning:** due to a bug in all versions lower than `6.0` of the [`SensioFrameworkExtraBundle`](https://github.com/sensiolabs/SensioFrameworkExtraBundle) bundle,
+if you have it installed you will likely get an error during the execution of the command above in regards to the missing `Nyholm\Psr7\Factory\Psr17Factory`
+class. To workaround the issue, if you are not using the PSR-7 bridge, please change the configuration of that bundle as follows:
 
-Add your DSN to `app/config/config.yml`:
+```yaml
+sensio_framework_extra:
+   psr_message:
+      enabled: false
+```
+
+For more details about the issue see [https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710](https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710).
+</Alert>
+
+## Configure
+
+Add your DSN to `config/packages/sentry.yaml`:
 
-```yaml {filename:app/config/config.yml}
+```yaml {filename:config/packages/sentry.yaml}
 sentry:
   dsn: "___PUBLIC_DSN___"
 ```
 
-If you **are not** using Symfony Flex, you'll also need to enable the bundle in `app/AppKernel.php`:
+If you **are not** using Symfony Flex, you'll also need to enable the bundle in `config/bundles.php`:
+
+```php {filename:config/bundles.php}
+<?php
+
+return [
+    // ...
+    Sentry\SentryBundle\SentryBundle::class => ['all' => true],
+];
+```
+
+### Monolog Integration
+
+If you are using monolog, these additional configuration and services are needed to log the errors correctly:
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+    register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
+    dsn: "%env(SENTRY_DSN)%"
```

Comment:
```suggestion
```

This is needless for the example to work, so I would remove it. In the first part of the page this is already mentioned and I think that further example should just add things in addition to what is considered to be already present